portfolio/src/main/webapp/index.html

```diff
@@ -1,66 +0,0 @@
-<!DOCTYPE html>
```

Comment:
Don't delete this!

Reply:
This file is no longer on this branch so I have no way of editing it. <!DOCTYPE html> and the copyright message is still on all of my other branches though.